app/src/main/java/org/mozilla/reference/browser/library/bookmarks/ui/viewholders/BookmarkFolderViewHolder.kt

```diff
@@ -0,0 +1,23 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package org.mozilla.reference.browser.library.bookmarks.ui.viewholders
+
+import mozilla.components.concept.storage.BookmarkNode
+import org.mozilla.reference.browser.library.LibraryItemView
+import org.mozilla.reference.browser.library.SelectionHolder
+import org.mozilla.reference.browser.library.bookmarks.ui.BookmarkViewInteractor
+
+class BookmarkFolderViewHolder(
+    view: LibraryItemView,
+    interactor: BookmarkViewInteractor,
+    private val selectionHolder: SelectionHolder<BookmarkNode>
+) : BookmarkNodeViewHolder(view, interactor) {
+
+    override fun bind(item: BookmarkNode) {
+        // to-do
+    }
+}
```

Comment:
Do we really need 2 view holders, I think that just one that knows how to display two different things will be enough.

Reply:
I think it makes sense to have different view holders for different view types than having it done using `if-else` statement.

There's also a parent class called `BookmarkNodeViewHolder`, then it should be removed too, right?